hera_gpu/tests/speed_test_output.txt

```diff
@@ -0,0 +1,41 @@
+512 ANTS GPU TIME (FLOAT): 0.673274040222 s
```

Comment:
This is a nice output to have in case users don't have a GPU of their own. I think we could have something similar for the in-line speed test commentary currently in `redcal.py`.

Reply:
This is very outdated actually iirc. Having a text output saved for the speed test could be useful but this file is very out of date so should be deleted/overwritten. It would also be useful to include some metadata about the date/commit it was last run as well as the GPU it was run on.

Reply:
The speed test should be resolved eventually, but it should be it's own pull request.